quic/s2n-quic-core/src/inet/checksum.rs

```diff
@@ -401,13 +401,18 @@ mod tests {
         }
     }
 
+    // Reduce the length to 4 for Kani until
+    // https://github.com/model-checking/kani/issues/3030 is fixed
+    #[cfg(any(kani, miri))]
+    const LEN: usize = if cfg!(kani) { 4 } else { 32 };
```

Comment:
I just noticed that this changes the length for MIRI as well from 16 to 32. Is that OK, or should I revert it?

Reply:
I guess we'll see if the CI job times out :)

Reply:
Ah it already went through. Looks ok to me! https://github.com/aws/s2n-quic/actions/runs/7922350014/job/21629765794?pr=2128